editor/src/viewport_tools/tools/shared/transformation_cage.rs

```diff
@@ -18,11 +18,22 @@ pub struct SelectedEdges {
 	bottom: bool,
 	left: bool,
 	right: bool,
+	// Aspect ratio in the form: x : 1 = width : height
```

Comment:
I think of aspect ratio as `width / height` or `height / width` (usually the former). Could you translate this form to also include a `width / height` form? It's easier for me, and possibly some other future maintainers, to understand.

Reply:
If we want the height side of the ratio to be one then you obviously need to do `width/height` for the width side.

Reply:
Ah, that makes sense. Too much thinking for my 1 AM brain :)

Reply:
Should I state this explicitly in the comment?

Reply:
Maybe just say something like:
> Aspect ratio in the form of `width/height`, or in other words `x:1 = width:height`
That way future people (or future me) won't need to think through the logic when it's also a future 1 AM, haha.